src/sb_utils.R

```diff
@@ -29,4 +46,42 @@ sb_render_post_xml <- function(sb_id, ..., xml_file = NULL){
   
   sb_replace_files(sb_id = sb_id, xml_file)
   
-}
\ No newline at end of file
+}
+
+# Helper function to create a task_table for the files that need to be pushed to SB
+do_item_replace_tasks <- function(files, sb_id, task_yml, out_dir) {
+  # Define task table rows
+  tasks <- basename(files)
+  
+  # Define task table columns
+  sb_push <- scipiper::create_task_step(
+    step_name = 'push_file_to_sb',
+    target_name = function(task_name, step_name, ...){
+      sprintf("%s_pushed_to_sb", task_name)
+    },
+    command = function(task_name, ...){
+      sprintf("item_replace_files('%s', '%s')", sb_id, task_name)
+    } 
+  )
+  
+  # Create the task plan
+  task_plan <- create_task_plan(
+    task_names = tasks,
+    task_steps = list(sb_push),
+    final_steps = c('push_file_to_sb'),
+    add_complete = FALSE)
+  
+  # Create the task remakefile
+  final_target <- sprintf("%s/%s_files_uploaded.ind", out_dir, gsub(".yml", "", basename(task_yml)))
+  create_task_makefile(
+    task_plan = task_plan,
+    makefile = task_yml,
+    include = 'remake.yml',
+    packages = c('sbtools'),
```

Comment:
I think you need to include `scipiper` too, even though I think you get away with it because you are building the file w/ scipiper loaded. But since `combine_to_ind` is a scipiper function, should include that for completeness